---
title: "Detailed computational study of the active site of the hepatitis C viral rna polymerase to aid novel drug design"
collection: publications
category: scientific articles
permalink: /publication/2013-10-12-ns5b-binding-free-energy
excerpt: 'This study investigates potential inhibitors of Hep C NS5B via binding free energy calculations'
date: 2013-10-12
venue: 'Journal of Chemical Information and Modeling'
paperurl: 'https://pubs.acs.org/doi/10.1021/ci4003969'
# bibtexurl: 'https://academicpages.github.io/files/bibtex1.bib'
citation: 'Barakat, K. H., Law, J., <ins>Prunotto, A.</ins>, Magee, W. C., Evans, D. H., Tyrrell, D. L., ... & Houghton, M. (2013). Detailed computational study of the active site of the hepatitis C viral RNA polymerase to aid novel drug design. <i>Journal of chemical information and modeling</i>, 53(11), 3031-3043.'
---

The hepatitis C virus (HCV) RNA polymerase, NS5B, is a leading target for novel and selective HCV drug design. The enzyme has been the subject of intensive drug discovery aimed at developing direct acting antiviral (DAA) agents that inhibit its activity and hence prevent the virus from replicating its genome. In this study, we focus on one class of NS5B inhibitors, namely nucleos(t)ide mimetics. Forty-one distinct nucleotide structures have been modeled within the active site of NS5B for the six major HCV genotypes. Our comprehensive modeling protocol employed 287 different molecular dynamics simulations combined with the molecular mechanics/Poissonâ€“Boltzmann surface area (MM-PBSA) methodology to rank and analyze these structures for all genotypes. The binding interactions of the individual compounds have been investigated and reduced to the atomic level. The present study significantly refines our understanding of the mode of action of NS5B-nucleotide-inhibitors, identifies the key structural elements necessary for their activity, and implements the tools for ranking the potential of additional much needed novel inhibitors of NS5B.