---
title: "Extensive tissue-specific expression variation and novel regulators underlying circadian behavior"
collection: publications
category: scientific articles
permalink: /publication/2021-01-29-cry-mutation
excerpt: 'This study investigates genetic variation in circadian gene expression'
date: 2021-01-29
venue: 'Science Advances'
paperurl: 'https://www.science.org/doi/10.1126/sciadv.abc3781'
# bibtexurl: 'https://academicpages.github.io/files/bibtex1.bib'
citation: 'Litovchenko, M., Meireles-Filho, A. C., Frochaux, M. V., Bevers, R. P., Prunotto, A., Anduaga, A. M., ... & Deplancke, B. (2021). Extensive tissue-specific expression variation and novel regulators underlying circadian behavior. <i>Science advances</i>, 7(5), eabc3781.'
---

Natural genetic variation affects circadian rhythms across the evolutionary tree, but the underlying molecular mechanisms are poorly understood. We investigated population-level, molecular circadian clock variation by generating >700 tissue-specific transcriptomes of Drosophila melanogaster (w1118) and 141 Drosophila Genetic Reference Panel (DGRP) lines. This comprehensive circadian gene expression atlas contains >1700 cycling genes including previously unknown central circadian clock components and tissue-specific regulators. Furthermore, >30% of DGRP lines exhibited aberrant circadian gene expression, revealing abundant genetic variationâ€“mediated, intertissue circadian expression desynchrony. Genetic analysis of one line with the strongest deviating circadian expression uncovered a novel cry mutation that, as shown by protein structural modeling and brain immunohistochemistry, disrupts the light-driven flavin adenine dinucleotide cofactor photoreduction, providing in vivo support for the importance of this conserved photoentrainment mechanism. Together, our study revealed pervasive tissue-specific circadian expression variation with genetic variants acting upon tissue-specific regulatory networks to generate local gene expression oscillations.